android/src/main/java/ru/tinkoff/kotea/android/ViewModelStoreOwnerExt.kt

```diff
@@ -21,7 +20,7 @@ import kotlin.reflect.KProperty
  * but you can share [Store] across a screens by specifying the [sharedViewModelKey].
  */
 fun <T : Store<*, *, *>> storeViaViewModel(
-    dispatcher: CoroutineDispatcher = Dispatchers.Default,
+    coroutineContext: CoroutineContext = Dispatchers.Default,
```

Comment:
Changing parameter type is binary incompatible change, please add a deprecated overload with the old signature

Reply:
done